src/commands.ts

```diff
@@ -317,7 +309,36 @@ export class CosmosDBCommands {
         }
         return null;
     }
-
+    public static async dropDocDBDatabase(db: DocDBDatabaseNode, explorer: CosmosDBExplorer): Promise<void> {
+        const confirmed = await vscode.window.showWarningMessage("Are you sure to delete database: " + db.label + " and its collections?",
```

Comment:
Add 'you want' to the messages and instead of using a colon, wrap the label in quotes. Something like this:

`"Are you sure you want to delete database '" + db.label + "' and its collections?"`